app/views/catalog/_downloads.html.erb

```diff
@@ -29,8 +31,16 @@
           <%= link_to(download_text(@document[:dc_format_s]), document.direct_download[:download], 'contentUrl' => document.direct_download[:download], data: { download: 'trigger', download_type: 'direct', download_id: document[:layer_slug_s] }) %>
         </li>
         <% end %>
-        <% if document.download_types.present? %>
+        <% if document.download_types.present? || document.iiif_download.present? %>
         <li role="presentation" class="dropdown-header">Generated</li>
+        <% end %>
+        <% if document.iiif_download.present? %>
```

Comment:
I know the formatting in this file is already crazy.. mind indenting this block?
```erb
<% if document.iiif_download.present? %>
  <li>
    <%= link_to "Download JPG", iiif_jpg_url, download: 'trigger' %>
...
```